script/build.js

```diff
@@ -690,6 +691,40 @@ smith.use(redirect({
   '/education/apply-for-education-benefits/': '/education/apply/'
 }));
 
+/*
+Add nonce attribute with substition string to all inline script tags
+Convert onclick event handles into nonced script tags
+*/
+smith.use((files, metalsmith, done) => {
```

Comment:
Could we put this in a separate file? I think we should start doing that more, this file is pretty big.

Reply:
Yeah that file is a beast. I'm in favor breaking every plugin into its own module as tedious as that may be.

Reply:
I'm fine to move it. Is there a good place in the repo for this stuff to live?

Reply:
I don't think we've really defined a place for it - maybe a subdirectory, `build/metalsmith-plugins`?